sky/serve/core.py

```diff
@@ -325,8 +328,7 @@ def update(
         'Service controller is stopped. There is no service to update. '
         f'To spin up a new service, use {backend_utils.BOLD}'
         f'sky serve up{backend_utils.RESET_BOLD}',
-        non_existent_message='Service does not exist. '
-        'To spin up a new service, '
+        non_existent_message='To spin up a new service, '
```

Comment:
why change this?

Reply:
IDK. That's inherited from [27eff42](https://github.com/andylizf/skypilot/commit/27eff425092d6f936a6480918ca09390fb3a85ec)

Reply:
Lets revert it back ;)